fili-core/src/test/groovy/com/yahoo/bard/webservice/druid/model/query/GroupByQuerySpec.groovy

```diff
@@ -204,10 +208,12 @@ class GroupByQuerySpec extends Specification {
         //expected result
         String queryString1 = stringQuery(default: true)
 
-        String dataSrc = ("""{
-                                "type":"query",
-                                "query":""" + queryString1 + """
-                            }""").replaceAll(/\s/, "")
+        String dataSrc = """
+            {
+                "type":"query",
+                "query":""" + queryString1 + """
+            }
+        """
```

Comment:
Is there a reason why this doesn't use the $queryString1 notation instead of the String concatenation?